cmd/kcp/kcp.go

```diff
@@ -195,7 +193,7 @@ func main() {
 						}
 
 						if pullMode && pushMode {
-							log.Fatal("can't set --push_mode and --pull_mode")
+							klog.Fatal("can't set --push_mode and --pull_mode")
```

Comment:
heh, this shouldn’t even be fatal, just an err return (as any good cli command should)